### PR TITLE
Added tangent function to scicalc

### PR DESCRIPTION
**Pull request by joelbd**

### CODE REVIEW

**Comment by joelbd** [comment]
@wkropat Can you take a look at this please?

File: scicalc [hunk 1]
```diff
@@ -35,9 +35,16 @@ def add(*args):
 		sum += arg
 	return sum
 
+# Take an number input and return the tangent value for that number
+def tan(x):
+	"""Return the tangent values of x"""
+
+	return math.tan(x)
```
Comment (wkropat) [inline scicalc]: Please make sure units are in radians and not degrees.

**Comment by wkropat** [comment]
Please make sure to use radians and not degrees.

**Comment by joelbd** [comment]
@wkropat Fixed!

**Comment by wkropat** [comment]
I think that I need write access to your repository so that I can merge it.

**Pull request merged**